lib/plugins/postcss-flatten-import.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+var urllib = require('url');
+var Bluebird = require('bluebird');
+var postcss = require('postcss');
+var collapseCSS = require('../collapsers/css');
+
+var postcssFlattenImports = postcss.plugin('postcss-flatten-import', function (opts) {
+  return function (css) {
+    return Bluebird.try(function () {
+      var tasks = [];
+
+      css.walkAtRules('import', function (importRule) {
+        var matches = /(?:url\()?['"]?([^'"\)\s]*)['"]?(?:\))?(?:\s+(.+))?/gi.exec(importRule.params);
```

Comment:
Probably, I'll give it a shot. Think I might moved this regex from the old parser to here.